conformanceExamples/src/main/scala/src/main/scala/za/co/absa/enceladus/conformanceExamples/CustomRuleSample1.scala

```diff
@@ -0,0 +1,51 @@
+package src.main.scala.za.co.absa.enceladus.conformanceExamples
+
+import org.apache.spark.sql.SparkSession
+import src.main.scala.za.co.absa.enceladus.conformanceExamples.interpreter.rules.custom.UppercaseCustomConformanceRule
+import za.co.absa.enceladus.conformance.CmdConfig
+import za.co.absa.enceladus.conformance.interpreter.DynamicInterpreter
+import za.co.absa.enceladus.dao.{EnceladusDAO, EnceladusRestDAO}
+import za.co.absa.enceladus.model.Dataset
+
+object CustomRuleSample1 {
+
+  case class ExampleRow(id: Int, makeUpper: String, leave: String)
+  case class OutputRow(id: Int, makeUpper: String, leave: String, doneUpper:String)
+
+  implicit val spark: SparkSession = SparkSession.builder().master("local[*]").appName("CustomRuleSample1")
+    .config("spark.sql.codegen.wholeStage", value = false)
+    .getOrCreate()
+
+  def main(args: Array[String]) {
```

Comment:
I would like this to be more of an example of how to use the DCE as a library inside a simple spark job. Have a look at [Atum](https://github.com/AbsaOSS/atum/blob/master/examples/src/main/scala/za/co/absa/atum/examples/SampleMeasurements1.scala) or [ABRiS](https://github.com/AbsaOSS/ABRiS/blob/master/src/main/scala/za/co/absa/abris/examples/KafkaAvroReader.scala).

Reply:
The idea here is to have the examples as runnable spark jobs.